docs/self-managed/setup/upgrade.md

```diff
@@ -2,130 +2,61 @@
-  --set identityKeycloak.auth.managementPassword=$KEYCLOAK_MANAGEMENT_SECRET \
-  --set identityKeycloak.postgresql.auth.password=$POSTGRESQL_SECRET
+helm search repo camunda/camunda-platform --versions
 ```
 
-:::note
+### Helm CLI version
 
-If you have specified on the first installation certain values, you have to specify them again on the upgrade either via `--set` or the values file and the `-f` flag.
+Use the recommended Helm CLI version for your Helm chart when upgrading. The Helm CLI version for each chart can be found on the [chart version matrix](https://helm.camunda.io/camunda-platform/version-matrix/).
 
-:::
+## Update your configuration
 
-For more details on the Keycloak upgrade path, you can also read the [Keycloak Upgrading Guide](https://www.keycloak.org/docs/latest/upgrading/).
+Configuration adjustments may be required when upgrading to a new version of the Helm chart. Before beginning your upgrade, ensure you have implemented any changes required by your new version.
 
-## Version update instructions
+<Tabs groupId="upgrades" defaultValue="8.5" queryString values={
+[
+{label: 'From Camunda 8.4 to 8.5', value: '8.5', },
+{label: 'From Camunda 8.3 to 8.4', value: '8.4', },
+{label: 'From Camunda 8.2 to 8.3', value: '8.3', },
+]
+}>
 
-As of the 8.4 release, the Camunda 8 **Helm chart** version is decoupled from the version of the application. The Helm chart release still follows the applications release cycle, but it has an independent version. (e.g., in the application release cycle 8.4, the chart version is 9.0.0).
+<TabItem value='8.5'>
 
-For more details about the applications version included in the Helm chart, review the [full version matrix](https://helm.camunda.io/camunda-platform/version-matrix/).
-
-You can also view all chart versions and application versions via Helm CLI as follows:
-
-```shell
-helm search repo camunda/camunda-platform --versions
-```
-
-**After applying the instructions for each Helm chart, get back to the top of this page and start the upgrade process. **
-
-## Helm CLI version
-
-For a smooth upgrade, always use the same Helm CLI version corresponding with the chart version that shows in the [chart version matrix](https://helm.camunda.io/camunda-platform/version-matrix/).
-
-## From Camunda 8.4 to 8.5
-
-### Helm chart 10.2.0+
+<h3>Helm chart 10.2.0+</h3>
 
 As of this Helm chart version, the image tags for all components are independent, and do not reference the global image tag. The value of the key `global.image.tag` is `null`, and each component now sets its own version.
 
 With this change, Camunda applications no longer require a unified patch version. For example, a given installation may use Zeebe version 8.5.1, and Operate version 8.5.2. Note that only the patch version can differ between components.
 
 The key `global.image.tag` is deprecated and it will be removed in the Camunda 8.6 release.
 
-### Helm chart 10.0.2+
+<h3>Helm chart 10.0.2+</h3>
```

Comment:
```suggestion
<h3>Helm Chart 10.0.2+</h3>
```

Reply:
Another thing I think is probably just not consistent/we can add to the cheatsheet - to keep **chart** lowercase.